helpdesk_mgmt/controllers/myaccount.py

```diff
@@ -108,7 +108,7 @@ def portal_my_ticket(self, ticket_id=None, **kw):
         except AccessError:
             return request.redirect('/my')
         values = self._ticket_get_page_view_values(ticket_sudo, **kw)
-        return request.render("helpdesk.portal_helpdesk_ticket_page", values)
+        return request.render("helpdesk_mgmt.portal_helpdesk_ticket_page", values)
```

Comment:
helpdesk_mgmt/controllers/myaccount.py:111:80: E501 line too long (82 > 79 characters)

Reply:
Added a commit for lint : https://github.com/OCA/helpdesk/pull/43/commits/630ee9e33dd46822f6ebfd49f057f1a0f5e32ab5